.github/workflows/android.yml

```diff
@@ -0,0 +1,28 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Why only for pushes to `master`?
This workflow didn't trigger for a pull request such as this, so there's probably something you'd want to change

Reply:
Thank you for the quick response! 
I actually have a question. My aim was to create an android.yml but how come the check was for the Arduino CI instead and no checks for android CI ? Is it because we still don't have an android app in our repository?

Reply:
No it's because you are triggering it in very specific ways. I suggest replacing the `on` value with [ push, pull_request]`.